paimon-common/src/main/java/org/apache/paimon/utils/RoaringBitmap32.java

```diff
@@ -86,7 +86,7 @@ public void serialize(DataOutput out) throws IOException {
     }
 
     public void deserialize(DataInput in) throws IOException {
-        roaringBitmap.deserialize(in);
+        roaringBitmap.deserialize(in, null);
```

Comment:
Maybe we can tested `roaringBitmap.deserialize(ByteBuffer bbf)` directly?  We actually know the exact byte count of the bitmap

Reply:
Did you mean adding a benchmark?

Reply:
From the test results, it is more optimal, and I have checked other formats, which also use it. we can just use it.

Reply:
The premise of ByteBuffer is that we need to know the actual size of the Bitmap. This seems to require recording the actual size at serialisation time.

Reply:
We can using it in delection vector bitmap and rbm aggregate function, but the bitmap and bsi index can not for now.

Reply:
> roaringBitmap.deserialize(ByteBuffer bbf)

Does the bitmap index not record the byte size? Would you be willing to integrate `roaringBitmap.deserialize(ByteBuffer bbf)` into a usable place such as dv?